backend/README.md

```diff
@@ -1,90 +1,78 @@
-• apidoc@0.22.1
-• babel-jest@24.9.0
-• bcryptjs@2.4.3
-• body-parser@1.19.0
-• cors@2.8.5
-• crypto@1.0.1
-• eslint@6.8.0
-• eslint-config-prettier@6.15.0
-• eslint-plugin-prettier@3.3.1
-• express@4.17.1
-• express-fileupload@1.2.0
-• generate-password@1.5.1
-• jest@24.9.0
-• jsonwebtoken@8.5.1
-• morgan@1.10.0
-• mysql2@2.3.3
-• nodemon@1.19.4
-• pg@7.18.2
-• pg-hstore@2.3.3
-• pre-commit@1.2.2
-• prettier@1.19.1
-• sequelize@5.22.3
-• sequelize-cli@5.5.1
-• supertest@4.0.2
-• swagger-jsdoc@3.7.0
-• swagger-ui-express@4.1.6
-```
-
-## 1. Start a local MySQL DB server
-
-Start a MySQL DB server on your machine with the following configuration
+# Quick-Start Guide
+This is a quick start guide for setting up the backend database of Updog
 
-`username: 'updogDev'`
+This guide assumes you have already cloned the repository and installed Node.js v14.x or later
 
-`password: 'password'`
+# 1. Install and set up MySQL
+1. [Install MySQL here](https://dev.mysql.com/downloads/installer/)
+
+2. In the "Choosing a Setup Type" page of the installer, select "Developer Default"
+
+3. Click "next" (using default settings) until you reach the "Accounts and Roles" page
+
+4. Add a root password for yourself
 
-`database: 'updog'`
+5. Click "Add User" and create a user with username "updogDev" and password "password"
 
-`host: '127.0.0.1'`
+6. Continue to click "next" or "execute" until the install process is done
 
-`port: 3306`
+# 2. Check that your local MySQL server is working correctly
+1. In MySQL Workbench, click Database->Manage Connections
+2. Click "Test Connection" and input the root password you set up earlier if prompted
+3. If successful, a popup similar to below should appear:
+![Test connection success popup](../readme-images/backend-mysql-test-connection-success.PNG)
 
-The config on the backend is already done, so once you get the db running locally it should be able to connect
+# 3. Create a new connection
+1. Create a new connection with the following settings:
+
+`username: 'updogDev'`
+
+`password: 'password'`
 
-NOTE: You do not need to create any tables, we just need a working db server
+`connection name: 'updog'`
 
-## 2. Install dependencies
+2. Click the home icon, and open the connection you just made
 
-`npm install`
+3. Under "Users and Privileges"->"Administrative Roles", check that the updogDev user you created has the DBA (Database Admin) permission - if you followed the steps in the [installer section](#1-install-and-set-up-mysql) you should already have this
 
-## 3. Migrate models to DB so tables would be created
+4. Click the database icon on the toolbar to create a new schema with the name "updog", all settings default, and click "Apply"
 
-To migrate the tables to the database, run the following commands:
+# 4. Create tables
 
-`npm install -g sequelize-cli` (only needs to be executed the first time)
+1. If you haven't already, execute `npm install` from within the `backend/` folder, then `npm install -g sequelize-cli`
```

Comment:
Might be worth mentioning to npm install in main /Updog/ too for consistency, since it's mentioned in FE readme